sass/_theme.scss

```diff
@@ -26,8 +26,8 @@ $highlights: (
   "doctag": "red",
   "meta": "peach",
   "section": "blue",
-  "tag": "subtext0",
-  "name": "mauve",
+  "tag": "teal",
+  "name": "blue",
```

Comment:
```suggestion
  "name": "mauve",
```

For the sake of contrast, I'd keep this `mauve` even if it goes against our style guide. Would be good to add a comment detailing why changing it to blue isn't good.

Same with `attr` below since that shouldn't be changed either.

Reply:
I kind of like that idea of using lavender or something slightly different from blue but at least not mauve 🤔

Reply:
![CleanShot 2024-09-08 at 14 41 25](https://github.com/user-attachments/assets/5e8c03cc-f5e2-4678-8fd0-6229264f61c5)
![CleanShot 2024-09-08 at 14 41 15](https://github.com/user-attachments/assets/cf5a8e04-3805-4c63-8992-59718b9354b0)

Reply:
Updated!